src/Admin/Ajax.php

```diff
@@ -117,7 +117,7 @@ public function handle_refresh_keys() {
 
     public function check_live_permissions() {
 		try{
-			$account = Authentication::getAccount(new Payplug($_POST['livekey']));
+			$account = Authentication::getAccount(new Payplug(PayplugWoocommerceHelper::get_live_key()));
```

Comment:
this could have a problem!
what if you've authorization to use this in test mode, but not in prod?
this won't allow you to use it right?!

Reply:
as we seen and discussed, it is not required to have the live key for test mode. Good to merge